chia/wallet/wallet_coin_store.py

```diff
@@ -139,6 +139,22 @@ async def get_unspent_coins_for_wallet(self, wallet_id: int) -> Set[WalletCoinRe
             )
         return set(self.coin_record_from_row(row) for row in rows)
 
+    async def get_all_unspent_coins(self) -> Set[WalletCoinRecord]:
+        """Returns set of CoinRecords that have not been spent yet for a wallet."""
+        async with self.db_wrapper.reader_no_transaction() as conn:
+            rows = await conn.execute_fetchall(
+                "SELECT * FROM coin_record WHERE spent_height=0"
+            )
+        return set(self.coin_record_from_row(row) for row in rows)
+
+    async def get_all_coins(self) -> Set[WalletCoinRecord]:
```

Comment:
why is this method added?

Reply:
The method used to exist. It was removed when the wallet coin store cache was removed. I'm using it in one of my tests. I could remove it, but the test wouldn't be as comprehensive.

Reply:
yeah we should remove it. If you want, you can make db queries manually from the test instead

Reply:
This is complete in my latest commit.